src/yb/tools/yb-admin-test.cc

```diff
@@ -538,6 +538,21 @@ TEST_F(AdminCliTest, TestGetClusterLoadBalancerState) {
   ASSERT_NE(output.find("ENABLED"), std::string::npos);
 }
 
+TEST_F(AdminCliTest, TestModifyPlacementPolicy) {
+  BuildAndStart();
+
+  // Modify the cluster placement policy to consist of 2 zones.
+  ASSERT_OK(CallAdmin("modify_placement_info", "c.r.z0,c.r.z1:2", 2, ""));
```

Comment:
Why 2 here? Does this test even work? I would assume we need a value >= 3 here. We have a validation check somewhere that the RF needs to be >= sum(min_num_replicas of all placement blocks)

Reply:
Just checked the code, we don't have any such validation in the `modify_placement_info` code path. Can you add a check in the below function that the `total replica count <= num_replicas specified`?

Reply:
Sure